doc/api_base.rst

```diff
@@ -275,3 +275,43 @@
         base.download_packages(base.transaction.install_set, progress)
         print("Installing...")
         base.do_transaction()
+
+  .. method:: fill_sack_from_repos_in_cache(load_system_repo=True)
+
+    Prepare Sack and Goal objects and load all enabled repositories from cache only, it doesn't download anything and it doesn't check if metadata are expired. If there is not enough metadata present (repond.xml or both primary.xml and solv file are missing) given repo is either skipped or it throws a :exc:`dnf.exceptions.RepoError` exception depending on :attr:`dnf.conf.Conf.skip_if_unavailable` configuration.
```

Comment:
I suggest that the description should be enhanced. Please could you add something like. For successful load it requires repond.xml plus metadata (xml, yaml) or repond.xml and generated cache files like .solv, .solvx. Please could you add a test and describe behaviour for yamls and additional metadata like updateinfo and so on.

Reply:
Updated doc text.

I have also added two more tests for modular metadata and extended some test with updateinfo checks.